### PR TITLE
Arnav 2021AM10805 WEB

### PR DESCRIPTION
**Pull request by Ashaeyx**

### CODE REVIEW

File: tests/Ashaeyx_test.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    t.fillcolor(color)
+    t.penup()
+    t.forward(i+50)
+    t.pendown()
+    t.left(angle)
+    t.begin_fill()
+    t.circle(size)
+    t.end_fill()
+turtle.exitonclick()
+turtle.bgcolor("black")
```
Comment (shubham1206agra) [inline tests/Ashaeyx_test.py:22]: Wrong file. Please delete.

Comment (Ashaeyx) [reply tests/Ashaeyx_test.py]: i know , i added by mistake , how do i delete this sir ?

File: src/Freshers.js [hunk 2]
```diff
@@ -177,4 +177,26 @@ export const FRESHERS = [
             },
         ]
     },
+    {
+        entryNum: 805,
+        name: "Arnav Sharma",
+        image: '/GitAssignment2022/assets/2021AM10805.jpg',
```
Comment (shubham1206agra) [inline src/Freshers.js:183]: no jpg file in asset folder

Comment (Ashaeyx) [reply src/Freshers.js]: converted to png

**Comment by shubham1206agra** [comment]
You have commit image too.

**Comment by Ashaeyx** [comment]
Yes sir , now I have pushed the committed image in main rep

**Pull request merged**